resources/web/style/admonishment.pcss

```diff
@@ -0,0 +1,38 @@
+#guide .Admonishment {
+  position: relative;
+  font-size: 16px;
+
+  &:hover {
+    color: black;
+    transition: all 0.3s ease;
+    .Admonishment-detail {
+      z-index: 10000;
+      left: -5em;
+      opacity: 1;
+      color: #555;
+      transition: color 0.5s ease;
+    }
+  }
+
+  &-title, &-version {
+    font-family:Monaco, 'DejaVu Sans Mono', 'Bitstream Vera Sans Mono', 'Lucida Console', monospace!important;
+    font-size: 0.8em;
+    border-bottom: 1px dashed #00a9e5;
+  }
+  &-version--deprecated {
+    text-decoration: line-through;
```

Comment:
Continuing the line of thought above, what value does this have over a class like say, `u-strikethrough` or something.

Reply:
I don't think it has any beyond the "traditional" separation or style and markup. I'll see about adding those utilities before merging.